action.yml

```
@@ -69,30 +69,36 @@ runs:
     - name: Decrypt cached session file
       id: decrypt
       if: ${{ steps.restore-cache.outcome == 'success' }}
-      continue-on-error: true
       shell: bash
       run: |
+        # Default to no session.
+        echo "session_found=false" >> $GITHUB_OUTPUT
 
         # Verify that the encrypted session file was restored from cache.
-        test -s ${{ steps.configure-cache.outputs.path }}
+        if [ ! -s "${{ steps.configure-cache.outputs.path }}" ]; then
```

```suggestion
        if [[ ! -s "${{ steps.configure-cache.outputs.path }}" ]]; then
```
we're in bash, use bash.